source/manage/mmctl-command-line-tool.rst

```diff
@@ -23,9 +23,9 @@ mmctl Command Line Tool
   :target: https://mattermost.com/deploy
   :alt: Available for Mattermost Self-Hosted deployments.
 
-The mmctl tool is a remote CLI tool for Mattermost which is installed locally and uses the Mattermost API. Authentication is done with either login credentials or an authentication token. This mmctl tool replaces the `CLI <https://docs.mattermost.com/manage/command-line-tools.html>`__ from Mattermost v6.0.
+The mmctl tool is a remote CLI tool for Mattermost which is installed locally and uses the Mattermost API. Authentication is done with either login credentials or an authentication token. This mmctl tool is included from Mattermost v6.0, and it replaces the `CLI <https://docs.mattermost.com/manage/command-line-tools.html>`__. The mmctl tool can currently be used alongside the Mattermost CLI tool. The Mattermost CLI tool will be deprecated in a future release.
```

Comment:
In what context is mmctl remote?

Reply:
@isacikgoz - Can you clarify what's meant by mmctl is remote, please?

Reply:
From @mgdelacroix: "`mmctl` works using the mattermost REST API. It's an independent tool that can be run on any machine that can reach the mattermost server that we want to manage, as opposed to the old CLI that needed to run on the same machine of the server.

The exception to this is local mode, that requires the tool to run on the same machine of the server and doesn't require authentication precisely because of this."